endpoint/endpoint.go

```diff
@@ -26,6 +26,14 @@ const (
 	OwnerLabelKey = "owner"
 )
 
+// TTL is a structure defining the TTL of a DNS record
+type TTL struct {
+	// The value of the TTL
+	Value int64
+	// Whether or not the TTL value is configured
+	IsConfigured bool
```

Comment:
Exactly:

It's ok to be implicit here, because it's not ambiguous and we don't have to safe the value we can compute it.